src/lib/modules/console/index.ts

```diff
@@ -112,9 +114,24 @@ class Console {
       this.history.push(cmd);
       this.saveHistory();
     }
-    const pluginCmds = this.plugins.getPluginsProperty("console", "console");
-    for (const pluginCmd of pluginCmds) {
-      const pluginResult = pluginCmd.call(this, cmd, {});
+    const plugins = this.plugins.getPluginsProperty("console", "console");
+    const helpDescriptions = [];
+    for (const plugin of plugins) {
+      // New API
+      if (plugin.options.description) {
+        helpDescriptions.push(plugin.options);
+      }
+      if (plugin.options.matches) {
```

Comment:
I am not a fan of this part as it creates an API that cannot be used in all case.
I much prefer the function way only